okteto.yml

```diff
@@ -0,0 +1,15 @@
+build:
+  hello-world:
+    image: okteto/gradle:6.5
```

Comment:
this should be `okteto.dev/java-gradle-hello-world:1.0.0`.  `okteto/gradle:6.5` should be indev.hello-world.image